scripts/new_project.kts

```diff
@@ -4,7 +4,16 @@ object NewProject {
 
     private const val ARGUMENT_DELIMITER = "="
     private const val DOT_SEPARATOR = "."
+    private const val HELP_MESSAGE = """
+Run kscript new_project.kts to create a new project with the following arguments:
+    package-name=   New package name (i.e. com.example.package)
+    app-name=       New app name (i.e. MyApp, "My App")
+    --help          Display usage message and exit
```

Comment:
Do we need to show this 👇 when running script `--help` 🤔  Is that mean we will show(suggestion) all commands that can be run?
```suggestion
    --help          Display usage message and exit
```

Reply:
I think @kaungkhantsoe was following the previous output message: 

<img width="715" alt="Screen Shot 2022-07-29 at 3 46 37 PM" src="https://user-images.githubusercontent.com/18277915/182120332-581c29ab-a228-4e28-bdb9-f9b620f0a86f.png">

I'm totally fine to remove it, as it doesn't bring much value 🚀

Reply:
I'm fine to keep it too, just asked for sure 🙏